packages/twenty-website/src/content/developers/self-hosting/self-hosting-var.mdx

```diff
@@ -37,6 +37,7 @@ yarn command:prod cron:calendar:calendar-event-list-fetch
     ['REDIS_URL', 'redis://localhost:6379', 'Redis connection url'],
     ['FRONT_BASE_URL', 'http://localhost:3001', 'Url to the hosted frontend'],
```

Comment:
this will be renamed FRONT_DOMAIN

Reply:
redirect (workspace.subDomain . FRONT_DOMAIN)

Reply:
Let's keep: FRONT_BASE_URL and extract defaultSubDomain + protocol + domain from there when we need it

Reply:
in case we are not in multidomain, we should expect defaultSubDomain to be extratable from FRONT_BASE_URL